src/components/MarketingCard/index.tsx

```diff
@@ -4,45 +4,21 @@ import {SFC} from 'types/generic';
 import {SocialMedia} from 'types/social-media';
 import {socialMediaDescriptions, socialMediaHandles, socialMediaUrls} from 'utils/social-media';
 
-import DiscordLogo from 'assets/logos/Discord.png';
-import FacebookLogo from 'assets/logos/Facebook.png';
-import GitHubLogo from 'assets/logos/GitHub.png';
-import InstagramLogo from 'assets/logos/Instagram.png';
-import LinkedInLogo from 'assets/logos/LinkedIn.png';
-import PinterestLogo from 'assets/logos/Pinterest.png';
-import RedditLogo from 'assets/logos/Reddit.png';
-import TNBGamingLogo from 'assets/logos/TNBGaming.png';
-import TwitchLogo from 'assets/logos/Twitch.png';
-import TwitterLogo from 'assets/logos/Twitter.png';
-import YouTubeLogo from 'assets/logos/YouTube.png';
+import {MARKETING_CARD_CONFIG} from './config';
 
 import * as S from './Styles';
+import './sprite.scss';
 
 export interface MarketingCardProps {
   customLink?: string;
   website: SocialMedia;
 }
 
 const MarketingCard: SFC<MarketingCardProps> = ({customLink, website}) => {
-  const renderImage = (src: any) => <S.Image alt={website} data-testid="MarketingCard__img" src={src} />;
-
-  const images = {
```

Comment:
removed this ineffecient implementation, as every call to `MarketingCard` will render all logo images, whereas we can implement it such that each marketing card renders a single image